Week3/homework/7-step3.js

```diff
@@ -23,3 +23,10 @@ console.log(y);
 
 // Add your explanation as a comment here
 
+/*
+
+The answer is a bit simple to be honest, when you log x, of course it will log a value which is number 9, but in the case of y is a bit different because the value is an object, this you can see by logging the typeof x and y,
+
+this happens because in Javascript, data types like numbers strings and booleans always pass by values, and objects pass by references
+
```

Comment:
Hi @SignosRk - your explanation is correct. However, there is one thing which I wanted to add. **JavaScript is always call by value. However, in case of objects, the value which is passed is the reference itself.** It doesn't change anything in the way you have learnt things as they behave according to your expectations but just wanted to clarify it.